.github/workflows/skyux-ci.yml

```diff
@@ -14,13 +14,13 @@ jobs:
   ci:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 2
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
           node-version-file: '.nvmrc'
-      - uses: blackbaud/skyux-sdk-actions@ng-14
+      - uses: blackbaud/skyux-sdk-actions@skip-install-deps-for-speedy-browser-set
```

Comment:
Chrome is preinstalled for GitHub Actions. This change skips the `playwright install-deps` when only using Chrome.